<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroupChat</class>
 <widget class="QWidget" name="GroupChat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>610</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>630</width>
    <height>610</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>630</width>
    <height>610</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*全局样式表*/
QPushButton{
	border:none;
}

/*局部样式表Top*/
QWidget#Top{
	background-color: rgb(231, 234, 247);
	border-bottom:1px solid rgb(149, 149, 149);
}

QPushButton#close_Btn:hover{
	background-color: rgb(255, 0, 0);
}

QPushButton#reduce_Btn:hover{
	background-color: rgb(217, 217, 217);
}

QGroupBox#groupBox{
	border: 1px solid red;
}

/*局部样式表Centet*/
QWidget#Center{
	background-color: rgb(231, 234, 247);
	border-bottom:1px solid rgb(149, 149, 149);
}

QTextBrowser#text_Browser {
    background-color: rgb(231, 234, 247);
    border-right: none;
}

QLabel#groupSum_Lab{
	background-color: rgb(240, 250, 250);
}

QListWidget#groupFriend_List{
	background-color: rgb(231, 234, 247);
}

QPushButton#close_Btntop:hover{
	background-color: rgb(255, 0, 0);
}

QPushButton#reduce_Btn:hover{
	background-color: rgb(217, 217, 217);
}

QListWidget#listWidget{
	background-color: rgb(231, 234, 247);
}
/*局部样式表Bottom*/
QWidget#Bottom{
	background-color: rgb(231, 234, 247);
}

QTextEdit#input_Edit{
	background-color: rgb(231, 234, 247);
	border:noe;
}

QPushButton#emote_Btn:hover{
	 border:1px solid rgb(203, 203, 203);
}

QPushButton#file_Btn:hover{
	 border:1px solid rgb(203, 203, 203);
}


QPushButton#close_Btnbottom{
	background-color: rgb(6, 157, 213);
	color: rgb(255, 255, 255);
	border-radius:5px;
	font:normal 18px &quot;楷体&quot;;
}

QPushButton#close_Btnbottom:hover{
	background-color: rgb(75,202,255);
}

QPushButton#sending_Btn{
	background-color: rgb(6, 157, 213);
	color: rgb(255, 255, 255);
	border-radius:5px;
	font:normal 18px &quot;楷体&quot;;
}

QPushButton#sending_Btn:hover{
	background-color: rgb(75,202,255);
}</string>
  </property>
  <widget class="QWidget" name="Top" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>630</width>
     <height>70</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>630</width>
     <height>70</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>630</width>
     <height>70</height>
    </size>
   </property>
   <widget class="QPushButton" name="reduce_Btn">
    <property name="geometry">
     <rect>
      <x>588</x>
      <y>0</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../../image/Image.qrc">
      <normaloff>:/new/icons/icons/becurrent/minus.png</normaloff>:/new/icons/icons/becurrent/minus.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="close_Btn">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>0</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../../image/Image.qrc">
      <normaloff>:/new/icons/icons/becurrent/close.png</normaloff>:/new/icons/icons/becurrent/close.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="headImage_Lab">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../image/Image.qrc">:/new/user/user/touxiang01.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="groupName_Lab">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>150</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>群聊名称</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>475</x>
      <y>25</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>150</width>
      <height>40</height>
     </size>
    </property>
    <property name="title">
     <string>ID</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QLabel" name="groupID">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>150</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>账号</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="Center" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>630</width>
     <height>380</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>630</width>
     <height>380</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>630</width>
     <height>380</height>
    </size>
   </property>
   <widget class="QLabel" name="groupSum_Lab">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>0</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>160</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>160</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>群成员：</string>
    </property>
   </widget>
   <widget class="QListWidget" name="groupFriend_List">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>20</y>
      <width>160</width>
      <height>360</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>160</width>
      <height>360</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>160</width>
      <height>360</height>
     </size>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>470</width>
      <height>380</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>470</width>
      <height>380</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>470</width>
      <height>380</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="Bottom" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>450</y>
     <width>630</width>
     <height>160</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>630</width>
     <height>160</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>630</width>
     <height>160</height>
    </size>
   </property>
   <widget class="QTextEdit" name="input_Edit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>630</width>
      <height>120</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>630</width>
      <height>120</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>630</width>
      <height>120</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="close_Btnbottom">
    <property name="geometry">
     <rect>
      <x>435</x>
      <y>130</y>
      <width>90</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>90</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>90</width>
      <height>25</height>
     </size>
    </property>
    <property name="text">
     <string>关闭</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sending_Btn">
    <property name="geometry">
     <rect>
      <x>535</x>
      <y>130</y>
      <width>90</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>90</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>90</width>
      <height>25</height>
     </size>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../../image/Image.qrc"/>
 </resources>
 <connections/>
</ui>
